packages/suite/src/components/firmware/ReconnectDevicePrompt.tsx

```diff
@@ -167,82 +176,67 @@ interface Props {
-                    </Heading>
-                    {!reconnectedInRequestedMode ? (
-                        text.steps.map((step, i) => (
-                            // static array, using index as a key is fine
-                            // eslint-disable-next-line react/no-array-index-key
-                            <React.Fragment key={i}>
+            {rebootMethod === 'automatic' &&
+            (rebootPhase === 'wait-for-confirm' || rebootPhase === 'disconnected') &&
+            device ? (
+                <ConfirmAction device={device} />
+            ) : (
+                <Wrapper data-test={`@firmware/reconnect-device/${requestedMode}`}>
+                    <StyledDeviceAnimation
+                        type="BOOTLOADER"
+                        size={200}
+                        shape="ROUNDED"
+                        device={expectedDevice}
+                        loop
+                    />
+                    <Content>
+                        <Heading>
+                            <HeadingText requestedMode={requestedMode} phase={rebootPhase} />
+                        </Heading>
+                        {rebootPhase !== 'done' ? (
+                            <>
                                 {/* First step asks for disconnecting a device */}
+                                <ReconnectStep
+                                    order={1}
+                                    active={rebootPhase !== 'disconnected'}
+                                    dataTest="@firmware/disconnect-message"
+                                >
+                                    <Translation id="TR_DISCONNECT_YOUR_DEVICE" />
+                                </ReconnectStep>
                                 {/* Second step reconnect in normal mode or bootloader */}
-                                <BulletPointWrapper>
-                                    <BulletPointNumber active={isStepActive(i)}>
-                                        {i + 1}
-                                    </BulletPointNumber>
-                                    <BulletPointText
-                                        active={isStepActive(i)}
-                                        data-test={isStepActive(i) ? step.dataTest : undefined}
+                                <ReconnectStep
+                                    order={2}
+                                    active={rebootPhase === 'disconnected'}
+                                    dataTest={`@firmware/connect-in-${requestedMode}-message`}
+                                >
+                                    <ReconnectLabel requestedMode={requestedMode} device={device} />
+                                </ReconnectStep>
+                                {/* TODO when this should be displayed? Previous step '2' was nonexistent. */}
+                                {false && showWebUSB && <WebusbButton />}
```

Comment:
Hmm :thinking: you may need webusb button in case you use webusb transport (this is obvious) && reconnected device is not paired with your browser. This might even happen when you are reconnecting from normal mode to bootloader mode, because from the browser's persective these are 2 different devices. Under normal conditions you won't hit this problem as your bootloader device is likely paired already. to test this you need to remove all device from chrome settings and try again.

Reply:
I've tried that exact use case (WebUSB + Unknown device + Firmware updating) and without that button, it wasn't possible to continue. I've displayed the button in [`ca313d29`](https://github.com/trezor/trezor-suite/pull/4077/commits/ca313d29605f7391b219c5c5a0c7155e6337fa70).